packages/shared/styles/components/atoms/SfCircleIcon.scss

```diff
@@ -1,78 +1,98 @@
 @import "../../helpers";
 
 .sf-circle-icon {
-  --button-padding: var(--spacer-big);
-  --button-border-radius: 100%;
+  --icon-color: var(--c-white);
+  --button-padding: var(--spacer-small);
+  --icon-size: 20px;
+  --button-size: 3.25rem;
   --button-background: var(--c-primary);
   --button-transition: box-shadow 300ms ease;
-
+  border-radius: 100%;
   position: var(--circle-icon-position, relative);
-  box-shadow: var(--circle-icon-box-shadow);
   &:hover {
-    --button-background: var(--c-primary-lighten);
-    --circle-icon-box-shadow: 0 0 0 0.3124rem rgba(var(--c-primary-base), 0.4);
+    --button-background: var(--c-primary);
+    --button-box-shadow: 0 0 0 0.3125rem rgba(var(--c-primary-base), 0.3);
   }
   &:active {
-    --circle-icon-box-shadow: 0 0 0 0.625rem rgba(var(--c-primary-bases), 0.4);
+    --button-transition: box-shadow 150ms linear;
+    --button-box-shadow: 0 0 0 0.5rem rgba(var(--c-primary-base), 0.3);
   }
   &:disabled {
-    --circle-icon-box-shadow: 0;
+    --button-box-shadow: 0;
+    --icon-color: var(--c-text-disabled);
+    &:hover {
+      --button-background: var(--c-light);
+    }
   }
+
   &__badge {
     position: absolute;
     top: var(--circle-icon-badge-top, 0);
     right: var(--circle-icon-badge-right, 0);
     box-sizing: border-box;
     min-width: var(
       --circle-icon-badge-width,
-      var(--circle-icon-badge-size, 1.1rem)
+      var(--circle-icon-badge-size, 1.125rem)
     );
     min-height: var(
       --circle-icon-badge-height,
-      var(--circle-icon-badge-size, 1.1rem)
+      var(--circle-icon-badge-size, 1.125rem)
     );
-    padding: var(--circle-icon-badge-padding, 0 3px);
     border-radius: var(--circle-icon-badge-border-radius, 0.625rem);
     background: var(--circle-icon-badge-background, var(--c-danger));
-    font: var(
+
+    @include font(
       --circle-icon-badge-font,
-      font(
-        --circle-icon-bage,
-        300,
-        var(--font-size-small),
-        1.6,
-        var(--body-font-family-secondary)
-      )
+      var(--font-weight-bold),
```

Comment:
Can you merge in  the newest changes and change variables names? It sould be `--font-bold` etc.